<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="NDI-LAP-339" name="Default test" tests="5" failures="4" timestamp="8 Apr 2014 05:47:52 GMT" time="5.759" errors="0">
  <testcase name="loginWithBlankCredentials" time="0.248" classname="testCases.LoginTC"/>
  <testcase name="loginWithBlankUsername" time="0.019" classname="testCases.LoginTC">
    <failure type="org.openqa.selenium.StaleElementReferenceException" message="Element not found in the cache - perhaps the page has changed since it was looked up
Command duration or timeout: 13 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: &amp;apos;2.39.0&amp;apos;, revision: &amp;apos;ff23eac&amp;apos;, time: &amp;apos;2013-12-16 16:11:15&amp;apos;
System info: host: &amp;apos;NDI-LAP-339&amp;apos;, ip: &amp;apos;172.16.10.45&amp;apos;, os.name: &amp;apos;Windows 7&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;6.1&amp;apos;, java.version: &amp;apos;1.7.0_25&amp;apos;
Session ID: 33928246-271d-4832-aa24-9ea760cd383c
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{platform=XP, acceptSslCerts=true, javascriptEnabled=true, cssSelectorsEnabled=true, databaseEnabled=true, browserName=firefox, handlesAlerts=true, browserConnectionEnabled=true, webStorageEnabled=true, nativeEvents=false, rotatable=false, locationContextEnabled=true, applicationCacheEnabled=true, takesScreenshot=true, version=28.0}]">
      <![CDATA[org.openqa.selenium.StaleElementReferenceException: Element not found in the cache - perhaps the page has changed since it was looked up
Command duration or timeout: 13 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: '2.39.0', revision: 'ff23eac', time: '2013-12-16 16:11:15'
System info: host: 'NDI-LAP-339', ip: '172.16.10.45', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0_25'
Session ID: 33928246-271d-4832-aa24-9ea760cd383c
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{platform=XP, acceptSslCerts=true, javascriptEnabled=true, cssSelectorsEnabled=true, databaseEnabled=true, browserName=firefox, handlesAlerts=true, browserConnectionEnabled=true, webStorageEnabled=true, nativeEvents=false, rotatable=false, locationContextEnabled=true, applicationCacheEnabled=true, takesScreenshot=true, version=28.0}]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:193)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:145)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:554)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:268)
	at org.openqa.selenium.remote.RemoteWebElement.clear(RemoteWebElement.java:113)
	at pageElements.LoginPage.enter_pswrd(LoginPage.java:31)
	at testCases.LoginTC.loginWithBlankUsername(LoginTC.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: org.openqa.selenium.remote.ErrorHandler$UnknownServerException: Element not found in the cache - perhaps the page has changed since it was looked up
Build info: version: '2.39.0', revision: 'ff23eac', time: '2013-12-16 16:11:15'
System info: host: 'NDI-LAP-339', ip: '172.16.10.45', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0_25'
Driver info: driver.version: unknown
	at <anonymous class>.fxdriver.cache.getElementAt(resource://fxdriver/modules/web_element_cache.js:7584)
	at <anonymous class>.Utils.getElementAt(file:///C:/Users/SAURAB~1.JAI/AppData/Local/Temp/anonymous5454375235636119740webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:7197)
	at <anonymous class>.fxdriver.preconditions.visible(file:///C:/Users/SAURAB~1.JAI/AppData/Local/Temp/anonymous5454375235636119740webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:8177)
	at <anonymous class>.DelayedCommand.prototype.checkPreconditions_(file:///C:/Users/SAURAB~1.JAI/AppData/Local/Temp/anonymous5454375235636119740webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:10814)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_/h(file:///C:/Users/SAURAB~1.JAI/AppData/Local/Temp/anonymous5454375235636119740webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:10831)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_(file:///C:/Users/SAURAB~1.JAI/AppData/Local/Temp/anonymous5454375235636119740webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:10836)
	at <anonymous class>.DelayedCommand.prototype.execute/<(file:///C:/Users/SAURAB~1.JAI/AppData/Local/Temp/anonymous5454375235636119740webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:10778)
]]>
    </failure>
  </testcase> <!-- loginWithBlankUsername -->
  <testcase name="loginWithBlankPassowrd" time="0.011" classname="testCases.LoginTC">
    <failure type="org.openqa.selenium.StaleElementReferenceException" message="Element not found in the cache - perhaps the page has changed since it was looked up
Command duration or timeout: 10 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: &amp;apos;2.39.0&amp;apos;, revision: &amp;apos;ff23eac&amp;apos;, time: &amp;apos;2013-12-16 16:11:15&amp;apos;
System info: host: &amp;apos;NDI-LAP-339&amp;apos;, ip: &amp;apos;172.16.10.45&amp;apos;, os.name: &amp;apos;Windows 7&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;6.1&amp;apos;, java.version: &amp;apos;1.7.0_25&amp;apos;
Session ID: 33928246-271d-4832-aa24-9ea760cd383c
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{platform=XP, acceptSslCerts=true, javascriptEnabled=true, cssSelectorsEnabled=true, databaseEnabled=true, browserName=firefox, handlesAlerts=true, browserConnectionEnabled=true, webStorageEnabled=true, nativeEvents=false, rotatable=false, locationContextEnabled=true, applicationCacheEnabled=true, takesScreenshot=true, version=28.0}]">
      <![CDATA[org.openqa.selenium.StaleElementReferenceException: Element not found in the cache - perhaps the page has changed since it was looked up
Command duration or timeout: 10 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: '2.39.0', revision: 'ff23eac', time: '2013-12-16 16:11:15'
System info: host: 'NDI-LAP-339', ip: '172.16.10.45', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0_25'
Session ID: 33928246-271d-4832-aa24-9ea760cd383c
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{platform=XP, acceptSslCerts=true, javascriptEnabled=true, cssSelectorsEnabled=true, databaseEnabled=true, browserName=firefox, handlesAlerts=true, browserConnectionEnabled=true, webStorageEnabled=true, nativeEvents=false, rotatable=false, locationContextEnabled=true, applicationCacheEnabled=true, takesScreenshot=true, version=28.0}]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:193)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:145)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:554)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:268)
	at org.openqa.selenium.remote.RemoteWebElement.clear(RemoteWebElement.java:113)
	at testCases.LoginTC.loginWithBlankPassowrd(LoginTC.java:50)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: org.openqa.selenium.remote.ErrorHandler$UnknownServerException: Element not found in the cache - perhaps the page has changed since it was looked up
Build info: version: '2.39.0', revision: 'ff23eac', time: '2013-12-16 16:11:15'
System info: host: 'NDI-LAP-339', ip: '172.16.10.45', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0_25'
Driver info: driver.version: unknown
	at <anonymous class>.fxdriver.cache.getElementAt(resource://fxdriver/modules/web_element_cache.js:7584)
	at <anonymous class>.Utils.getElementAt(file:///C:/Users/SAURAB~1.JAI/AppData/Local/Temp/anonymous5454375235636119740webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:7197)
	at <anonymous class>.fxdriver.preconditions.visible(file:///C:/Users/SAURAB~1.JAI/AppData/Local/Temp/anonymous5454375235636119740webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:8177)
	at <anonymous class>.DelayedCommand.prototype.checkPreconditions_(file:///C:/Users/SAURAB~1.JAI/AppData/Local/Temp/anonymous5454375235636119740webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:10814)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_/h(file:///C:/Users/SAURAB~1.JAI/AppData/Local/Temp/anonymous5454375235636119740webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:10831)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_(file:///C:/Users/SAURAB~1.JAI/AppData/Local/Temp/anonymous5454375235636119740webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:10836)
	at <anonymous class>.DelayedCommand.prototype.execute/<(file:///C:/Users/SAURAB~1.JAI/AppData/Local/Temp/anonymous5454375235636119740webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:10778)
]]>
    </failure>
  </testcase> <!-- loginWithBlankPassowrd -->
  <testcase name="loginWithWrongCredentials" time="0.014" classname="testCases.LoginTC">
    <failure type="org.openqa.selenium.StaleElementReferenceException" message="Element not found in the cache - perhaps the page has changed since it was looked up
Command duration or timeout: 13 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: &amp;apos;2.39.0&amp;apos;, revision: &amp;apos;ff23eac&amp;apos;, time: &amp;apos;2013-12-16 16:11:15&amp;apos;
System info: host: &amp;apos;NDI-LAP-339&amp;apos;, ip: &amp;apos;172.16.10.45&amp;apos;, os.name: &amp;apos;Windows 7&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;6.1&amp;apos;, java.version: &amp;apos;1.7.0_25&amp;apos;
Session ID: 33928246-271d-4832-aa24-9ea760cd383c
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{platform=XP, acceptSslCerts=true, javascriptEnabled=true, cssSelectorsEnabled=true, databaseEnabled=true, browserName=firefox, handlesAlerts=true, browserConnectionEnabled=true, webStorageEnabled=true, nativeEvents=false, rotatable=false, locationContextEnabled=true, applicationCacheEnabled=true, takesScreenshot=true, version=28.0}]">
      <![CDATA[org.openqa.selenium.StaleElementReferenceException: Element not found in the cache - perhaps the page has changed since it was looked up
Command duration or timeout: 13 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: '2.39.0', revision: 'ff23eac', time: '2013-12-16 16:11:15'
System info: host: 'NDI-LAP-339', ip: '172.16.10.45', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0_25'
Session ID: 33928246-271d-4832-aa24-9ea760cd383c
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{platform=XP, acceptSslCerts=true, javascriptEnabled=true, cssSelectorsEnabled=true, databaseEnabled=true, browserName=firefox, handlesAlerts=true, browserConnectionEnabled=true, webStorageEnabled=true, nativeEvents=false, rotatable=false, locationContextEnabled=true, applicationCacheEnabled=true, takesScreenshot=true, version=28.0}]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:193)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:145)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:554)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:268)
	at org.openqa.selenium.remote.RemoteWebElement.clear(RemoteWebElement.java:113)
	at pageElements.LoginPage.enter_usernm(LoginPage.java:26)
	at testCases.LoginTC.loginWithWrongCredentials(LoginTC.java:62)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: org.openqa.selenium.remote.ErrorHandler$UnknownServerException: Element not found in the cache - perhaps the page has changed since it was looked up
Build info: version: '2.39.0', revision: 'ff23eac', time: '2013-12-16 16:11:15'
System info: host: 'NDI-LAP-339', ip: '172.16.10.45', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0_25'
Driver info: driver.version: unknown
	at <anonymous class>.fxdriver.cache.getElementAt(resource://fxdriver/modules/web_element_cache.js:7584)
	at <anonymous class>.Utils.getElementAt(file:///C:/Users/SAURAB~1.JAI/AppData/Local/Temp/anonymous5454375235636119740webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:7197)
	at <anonymous class>.fxdriver.preconditions.visible(file:///C:/Users/SAURAB~1.JAI/AppData/Local/Temp/anonymous5454375235636119740webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:8177)
	at <anonymous class>.DelayedCommand.prototype.checkPreconditions_(file:///C:/Users/SAURAB~1.JAI/AppData/Local/Temp/anonymous5454375235636119740webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:10814)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_/h(file:///C:/Users/SAURAB~1.JAI/AppData/Local/Temp/anonymous5454375235636119740webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:10831)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_(file:///C:/Users/SAURAB~1.JAI/AppData/Local/Temp/anonymous5454375235636119740webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:10836)
	at <anonymous class>.DelayedCommand.prototype.execute/<(file:///C:/Users/SAURAB~1.JAI/AppData/Local/Temp/anonymous5454375235636119740webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:10778)
]]>
    </failure>
  </testcase> <!-- loginWithWrongCredentials -->
  <testcase name="loginWithCorrectCredentials" time="0.012" classname="testCases.LoginTC">
    <failure type="org.openqa.selenium.StaleElementReferenceException" message="Element not found in the cache - perhaps the page has changed since it was looked up
Command duration or timeout: 11 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: &amp;apos;2.39.0&amp;apos;, revision: &amp;apos;ff23eac&amp;apos;, time: &amp;apos;2013-12-16 16:11:15&amp;apos;
System info: host: &amp;apos;NDI-LAP-339&amp;apos;, ip: &amp;apos;172.16.10.45&amp;apos;, os.name: &amp;apos;Windows 7&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;6.1&amp;apos;, java.version: &amp;apos;1.7.0_25&amp;apos;
Session ID: 33928246-271d-4832-aa24-9ea760cd383c
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{platform=XP, acceptSslCerts=true, javascriptEnabled=true, cssSelectorsEnabled=true, databaseEnabled=true, browserName=firefox, handlesAlerts=true, browserConnectionEnabled=true, webStorageEnabled=true, nativeEvents=false, rotatable=false, locationContextEnabled=true, applicationCacheEnabled=true, takesScreenshot=true, version=28.0}]">
      <![CDATA[org.openqa.selenium.StaleElementReferenceException: Element not found in the cache - perhaps the page has changed since it was looked up
Command duration or timeout: 11 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: '2.39.0', revision: 'ff23eac', time: '2013-12-16 16:11:15'
System info: host: 'NDI-LAP-339', ip: '172.16.10.45', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0_25'
Session ID: 33928246-271d-4832-aa24-9ea760cd383c
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{platform=XP, acceptSslCerts=true, javascriptEnabled=true, cssSelectorsEnabled=true, databaseEnabled=true, browserName=firefox, handlesAlerts=true, browserConnectionEnabled=true, webStorageEnabled=true, nativeEvents=false, rotatable=false, locationContextEnabled=true, applicationCacheEnabled=true, takesScreenshot=true, version=28.0}]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:193)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:145)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:554)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:268)
	at org.openqa.selenium.remote.RemoteWebElement.clear(RemoteWebElement.java:113)
	at testCases.LoginTC.loginWithCorrectCredentials(LoginTC.java:74)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: org.openqa.selenium.remote.ErrorHandler$UnknownServerException: Element not found in the cache - perhaps the page has changed since it was looked up
Build info: version: '2.39.0', revision: 'ff23eac', time: '2013-12-16 16:11:15'
System info: host: 'NDI-LAP-339', ip: '172.16.10.45', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0_25'
Driver info: driver.version: unknown
	at <anonymous class>.fxdriver.cache.getElementAt(resource://fxdriver/modules/web_element_cache.js:7584)
	at <anonymous class>.Utils.getElementAt(file:///C:/Users/SAURAB~1.JAI/AppData/Local/Temp/anonymous5454375235636119740webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:7197)
	at <anonymous class>.fxdriver.preconditions.visible(file:///C:/Users/SAURAB~1.JAI/AppData/Local/Temp/anonymous5454375235636119740webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:8177)
	at <anonymous class>.DelayedCommand.prototype.checkPreconditions_(file:///C:/Users/SAURAB~1.JAI/AppData/Local/Temp/anonymous5454375235636119740webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:10814)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_/h(file:///C:/Users/SAURAB~1.JAI/AppData/Local/Temp/anonymous5454375235636119740webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:10831)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_(file:///C:/Users/SAURAB~1.JAI/AppData/Local/Temp/anonymous5454375235636119740webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:10836)
	at <anonymous class>.DelayedCommand.prototype.execute/<(file:///C:/Users/SAURAB~1.JAI/AppData/Local/Temp/anonymous5454375235636119740webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:10778)
]]>
    </failure>
  </testcase> <!-- loginWithCorrectCredentials -->
</testsuite> <!-- Default test -->
